### PR TITLE
Install knative locally on *nix by default

### PR DESCRIPTION
**Pull request by montaro**
See also #3017

### CODE REVIEW

**Comment by marcobebway** [comment]
I have verified the following locally on minikube:
- Install kyma without any feature flag installs knative (eventing, serving) by default.
- Install kyma with the `--disable-knative` flag, install kyma without knative.

**Comment by marcobebway** [comment]
I have verified the following locally on minikube:
- Install kyma without any flags, will install knative (eventing, serving) by default.
- Install kyma with the `--disable-knative` flag, will install kyma without knative.
> Note: In both scenarios kyma tests were green.

File: installation/cmd/run.ps1 [hunk 1]
```diff
@@ -13,15 +13,21 @@ $INSTALLER_EXTRA_ARGS = ""
...
 
-if ($KNATIVE -eq $true) {
+if ($DISABLE_KNATIVE -eq $false)
+{
     $MINIKUBE_EXTRA_ARGS = "${MINIKUBE_EXTRA_ARGS} -memory 10240 -disk_size 30g"
-    $INSTALLER_EXTRA_ARGS = "${INSTALLER_EXTRA_ARGS} -knative"
+}
+else
+{
+    $INSTALLER_EXTRA_ARGS = "${INSTALLER_EXTRA_ARGS} -disable-knative"
```
Comment (radufa) [inline installation/cmd/run.ps1]: It should be
```
$INSTALLER_EXTRA_ARGS = "${INSTALLER_EXTRA_ARGS} -disable_knative"
```

Comment (montaro) [reply installation/cmd/run.ps1]: OK

**Comment by abbi-gaurav** [comment]
This PR should be closed in favor of #3403. @montaro could you please close it.

**Comment by montaro** [comment]
Fixed in https://github.com/kyma-project/kyma/pull/3403

**Pull request closed without merging**